tools/mo/openvino/tools/mo/moc_frontend/pipeline.py

```diff
@@ -3,21 +3,19 @@
 
 import argparse
 import logging as log
-from typing import List
```

Comment:
Why changes in this file are needed? It seems they all are not related to this PR, please revert.

Reply:
Reverted.